docs/resource-navigation.md

```diff
@@ -1,64 +1,8 @@
-# Resource Navigation
-
-Resources can be loaded either from the file system or a configured cluster. 
-
-- Selecting the "Browse" button in the File Explorer prompts for a folder containing resource manifests.
-- Selecting "Show Cluster Objects" in the Cluster Explorer loads available resources using the configured kubeconfig variable. See [Cluster Preview](cluster-integration.md) for more details.
-
-Once selected, a folder the File Explorer and Navigator will be populated accordingly. Monokle will attempt to synchronize 
-automatically with any external file/folder changes made in the selected folder, but the Refresh button next to the Browse button
-allows for manual reloading of the folder, if needed.
-
-<!---## **File Explorer**
-
-- The File Explorer now shows all files parsed in the specified folder:
-  - Excluded files are greyed out.
-  - Files containing resource manifests have a number displayed after them, showing the number of 
-    resources found in that file.
-
-Selecting a file will highlight contained resources in the Navigator and attempt to scroll them into view. The contents of the 
-file will be shown in the source editor to the right - allowing for direct editing of the file.
-
-![File Explorer Navigation](img/resource-navigation-2-1.5.0.png)
-
-In this screenshot:
-
-- The argo-rollouts-aggregate-roles.yaml file is selected - the "3" after the filename indicates it contains 3 kubernetes resources.
-- The 3 ClusterRole resources in thie file are highlighed in the Navigator in the middle.
-- The contents of the file are shown in the editor to the right.
-
-While in the File Explorer, the File Action drop down list is available by clicking the ellipsis to the right of the file name.
-
-![Explorer File Actions](img/explorer-file-actions-1.5.0.png)
-
-In the File Actions pop up menu, the following actions are available:
-- Add Resource
-- Filter on this file
-- Add to Files: Exclude
-- Copy Path
-- Copy Relative Path
-- Rename
-- Delete
-- Reveal in Finder
-
-The same functionality is available for Folders:
-
-![Explorer File Actions](img/explorer-folder-actions-1.5.0.png)
-
-In the Folder Actions pop up menu, the following actions are available:
-- New Folder
-- New Resource
-- Filter on this folder
-- Add to Files: Exclude
-- Copy Path
-- Copy Relative Path
-- Rename
-- Delete
-- Reveal in Finder -->
+# Navigating Resources
 
 ## **Resource Navigator**
 
-The Navigator shows all resources found recursively in the selected folder. Resources are grouped into sections/subsections based
+The Navigator shows all resources found recursively in the folder selected in the File Explorer. Resources are grouped into sections/subsections based
```

Comment:
Suggest changing the first sentence to "The Navigator shows all resources found in the current project." ?